### ISSUE TITLE
Sunsetting ssbc/ssb-spec-drafts

### ISSUE THREAD

**Issue by cryptix** [ssbc/sips#3]
I appreciate this effort a lot!

I guess https://github.com/ssbc/ssb-spec-drafts should be sunset to avoid confusion at least. At least the content warning one should be carried over. It's the most adopted of them.. 😅.

As an related issue, I might have missed it but there is no "status" flag on these here, right? Like, "draft", "in review", "version 1".

**Comment by staltz** [comment]
> I guess https://github.com/ssbc/ssb-spec-drafts should be sunset to avoid confusion at least

Yup, will do! I looked at that one closely in the past few days. It would be good to transfer one or some of them here as well.

> As an related issue, I might have missed it but there is no "status" flag on these here, right? Like, "draft", "in review", "version 1".

This is something we discussed a lot at p2p-basel, and several of us were afraid of the risk of centralization by having an org that officially blesses specs, or orders them by maturity or something like that. So instead, we are opting to just list everything that is out there. This was mentioned in the README, below the table:

> This list is intentionally not categorizing SIPs by maturity, category, or status. We don't want to make judgement calls here, because that could cause centralization of decision power. Instead, look for curations and categorizations of SIPs among the SSB community.

**Comment by cryptix** [comment]
Okay, cool!

In that case I'm happy to carry over the GabbyGrove one myself.
